api/internal/plugins/utils/utils_test.go

```diff
@@ -86,8 +87,10 @@ func TestUpdateResourceOptions(t *testing.T) {
 	}
 	for i, c := range cases {
 		name := fmt.Sprintf("test%d", i)
-		in.Append(makeConfigMap(rf, name, c.behavior, c.hashValue))
-		expected.Append(makeConfigMapOptions(rf, name, c.behavior, !c.needsHash))
+		err := in.Append(makeConfigMap(rf, name, c.behavior, c.hashValue))
+		require.NoError(t, err)
```

Comment:
same here

Reply:
I see both styles in the codebase, actually. Please don't make me go are redo the work. Also, FWIW I find it more readable if the check is on a separate line.